### PR TITLE
Align documentation reference expander icons with labels texts.

### PR DESCRIPTION
**Pull request by mondeja**
## Before

leaflet-doc-before

## After

leaflet-doc-after

### CODE REVIEW

File: docs/docs/css/reference.css [hunk 1]
```diff
@@ -31,6 +31,7 @@ label {
 div.accordion > div.accordion-overflow > div.accordion-content {
     max-height: 0;
     display: none;
+    pointer-events: none;
```
Comment (johnd0e) [inline docs/docs/css/reference.css:34]: Why?

File: docs/docs/css/reference.css [hunk 2]
```diff
@@ -48,6 +49,7 @@ div.accordion.expanded > div.accordion-overflow > div.accordion-content {
 /* 		height: auto; */
     max-height: none;
     display: block;
+    pointer-events: fill;
```
Comment (johnd0e) [inline docs/docs/css/reference.css:52]: Why?

**Review by johnd0e — CHANGES_REQUESTED** [review]
In Chrome I see different effect, not like on your gifs.
1. Collapsed marker is not aligned
2. It moves vertically during expanding.

**Comment by mondeja** [comment]
Sorry, but I don't remember the details of these changes, so I'm closing this. Feel free to reopen and edit it if you see this pull useful.

**Pull request closed without merging**